docs/packages/markdown/loader.js

```diff
@@ -29,6 +29,44 @@ function moduleIDToJSIdentifier(moduleID) {
     .join('');
 }
 
+const componentPackageMapping = {
+  material: {},
+  base: {},
+};
+
+const packages = [
+  {
+    name: 'mui-material',
+    product: 'material',
+    paths: [
+      path.join(__dirname, '../../../packages/mui-lab/src'),
+      path.join(__dirname, '../../../packages/mui-material/src'),
+      path.join(__dirname, '../../../packages/mui-base/src'),
```

Comment:
Why is base here?

Reply:
because `@mui/base` is a dependency on material.

Reply:
This part is related on components only (neglect utilities, hooks or constants for now)